java/custom/org/mozilla/Test.java

```diff
@@ -0,0 +1,6 @@
+package org.mozilla;
```

Comment:
> Can you move this class in the bench/ directory? This way we don't need to package it in classes.jar for release.

How about I put it into the tests/ directory? Then it'll go into tests.jar, which gets added to the *config.jars* list when you run the benchmark (although the mechanism is indirect; I think it happens because the default *midletClassName* is `RunTests`, and tests.jar gets added to *config.jars* in that case).

This benchmark already relies on tests/native.js (which gets loaded if tests.jar is in *config.jars*), so there's precedence for a dependency on tests/. Maybe we should even move the whole bench/ directory into tests/ at some point.

In any case, I've made that change in e21ac5a15e515a43063139cd1e4574e1e0ac8edf. What do you think?